jsk_fetch_robot/fetcheus/fetch-interface.l

```diff
@@ -261,7 +261,8 @@ Example: (send self :gripper :position) => 0.00"
         (send self :remprop :go-pos-unsafe-no-wait-goal)
 
         ;; yaw-x-yaw approach
-        (ros::ros-info "go-pos-unsafe: (yaw, x, yaw) = (~A ~A ~A)" d0 d1 d2)
+        (ros::ros-info "go-pos-unsafe: (yaw, x, yaw) = (~A ~A ~A)"
+                       d0 (* (if forward-p 1.0 -1.0) d1) d2)
```

Comment:
I think this will show both
`go-pos-unsafe: (x y d) = ( ...) -> (yaw x yaw) = ( ... )`
and
`send-cmd-vel-raw: ....
```

Reply:
what are you considering about?
I dont understand what you are worrying about.